app/views/posts/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>posts display<h1>
\ No newline at end of file
+<%= render 'layouts/user_card', user: @user %>
+
+<% @posts.each do |post| %>
+  <%= render 'layouts/post_card', post: post %>
+  <%= render 'layouts/comments_card', post: post %>
+<% end %>
+<div class="flex">
+  <span class="text">
+      Showing <span class="show">1</span> to <span class="">2 posts</span> of <span class=" "><%= @user.posts_counter %></span> Entries
+  </span>
+  <div class="">
+    <button class="white">
```

Comment:
- [ ] `@post.return_recent_comments` is missing in your post _index_ view. As this activity requires that we use the `recent_comments` method defined in the _post_ model, in your case `return_recent_comments`, to render the user's 5 most recent comments in the `app/views/posts/index.html.erb` view, it should be similar to:

```ruby
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```